extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js

```diff
@@ -109,6 +109,9 @@ const DISABLED_ANCESTORS = [
   'amp-carousel',
 ].join(',');
 
+/** @type {Set<string>} */
```

Comment:
Add a comment that later this will include Picture, thus is a Set.

Or, just make this a comparison to `img` for now with a comment.